recommendation/system.web.json

```diff
@@ -116,21 +129,321 @@
               "Type": "Using",
               "Value": "System.Web",
               "Description": "Remove System.Web namespace.",
-              "ActionValidation":{"Contains":"", "NotContains":"using System.Web;"}
+              "ActionValidation": {
+                "Contains": "",
+                "NotContains": "using System.Web;"
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "Type": "Class",
+      "Name": "IHttpHandler",
+      "Value": "System.Web.IHttpHandler",
+      "KeyType": "BaseClass",
+      "ContainingType": "",
+      "RecommendedActions": [
+        {
+          "Source": "Amazon",
+          "Preferred": "Yes",
+          "TargetFrameworks": [
+            {
+              "Name": "netcoreapp3.1",
+              "TargetCPU": [ "x86", "x64", "ARM32", "ARM64" ]
+            },
+            {
+              "Name": "net5.0",
+              "TargetCPU": [ "x86", "x64", "ARM32", "ARM64" ]
+            }
+          ],
+          "Description": "Remove IHttpHandler in namespace System.Web and replace HttpContext in namespace Microsoft.AspNetCore.Http.",
+          "Actions": [
+            {
+              "Name": "RemoveDirective",
+              "Type": "Using",
+              "Value": "System.Web",
+              "Description": "Remove System.Web namespace.",
+              "ActionValidation": {
+                "Contains": "",
+                "NotContains": "using System.Web;"
+              }
+            },
+            {
+              "Name": "AddDirective",
+              "Type": "Using",
+              "Value": "Microsoft.AspNetCore.Http",
+              "Description": "Add Microsoft.AspNetCore.Http namespace.",
+              "ActionValidation": {
+                "Contains": "using Microsoft.AspNetCore.Http;",
+                "NotContains": ""
+              }
+            },
+            {
+              "Name": "AddDirective",
+              "Type": "Using",
+              "Value": "System.Threading.Tasks",
+              "Description": "Add System.Threading.Tasks namespace",
+              "ActionValidation": {
+                "Contains": "using System.Threading.Tasks;",
+                "NotContains": ""
+              }
+            },
+            {
+              "Name": "RemoveBaseClass",
+              "Type": "Class",
+              "Value": "IHttpHandler",
+              "Description": "Remove the base class IHttpHandler",
+              "ActionValidation": {
+                "Contains": "",
+                "NotContains": ":IHttpHandler"
+              }
+            },
+            {
+              "Name": "AddExpression",
+              "Type": "Class",
+              "Value": "RequestDelegate _next = null;",
+              "Description": "Add a new global variable to replace the OwinMiddleware global abstract variable of Next.",
+              "ActionValidation": {
+                "Contains": "RequestDelegate_next=null;",
```

Comment:
spacing might be an issue. Did you verify to make sure we remove all spaces before validating? I don't remember if that is the case. Otherwise this should be RequestDelaguate_next = null; instead.

Reply:
yes, it does remove spaces before validation

Reply:
Although we do not need spaces, let's be consistent and add them anyway for readability and consistency if that's ok.